apps/server/src/app/__tests__/utils/account.ts

```diff
@@ -35,7 +35,7 @@ export async function createTestInvestmentAccount(
         join(__dirname, `../test-data/${portfolio}/holdings.csv`)
     )
 
-    const [_deleted, ...securities] = await prisma.$transaction([
+    const [, ...securities] = await prisma.$transaction([
```

Comment:
I guess we can just keep this as it was since it's the only change in this file?

```suggestion
    const [_deleted, ...securities] = await prisma.$transaction([
```

Reply:
sure just removing an unused value so ide stops complaining, but ill put it back for now and remove in separate PR